src/main/res/values/donottranslate.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <string name="setting_verbose_enabled_key">setting_verbose_enabled_key</string>
```

Comment:
Why move these to **strings.xml**? If we translate to different languages, these don't need to be translated, that's why I prefer to keep them in a separate file.